### PR TITLE
Save all outputs of TSNR calculation, and fix python 3 unicode bug

### PR DESCRIPTION
**Pull request by jvarada**
#154

### CODE REVIEW

File: mriqc/reports/generators.py [hunk 1]
```diff
@@ -176,14 +178,14 @@ def summary_cover(dframe, qctype, failed=None, sub_id=None, out_file=None):
     # Format the size
     #pylint: disable=E1101
     newdf[['size_x', 'size_y', 'size_z']] = newdf[['size_x', 'size_y', 'size_z']].astype(np.uint16)
-    formatter = lambda row: ur'%d \u00D7 %d \u00D7 %d' % (
```
Comment (oesteban) [inline mriqc/reports/generators.py]: The u must go away, but the r must stay

File: mriqc/reports/generators.py [hunk 2]
```diff
@@ -176,14 +178,14 @@ def summary_cover(dframe, qctype, failed=None, sub_id=None, out_file=None):
...
+    formatter = lambda row: '%d \u00D7 %d \u00D7 %d' % (
         row['size_x'], row['size_y'], row['size_z'])
     newdf['size'] = newdf[['size_x', 'size_y', 'size_z']].apply(formatter, axis=1)
 
     # Format spacing
     newdf[['spacing_x', 'spacing_y', 'spacing_z']] = newdf[[
         'spacing_x', 'spacing_y', 'spacing_z']].astype(np.float32)  #pylint: disable=E1101
-    formatter = lambda row: ur'%.3f \u00D7 %.3f \u00D7 %.3f' % (
```
Comment (oesteban) [inline mriqc/reports/generators.py]: Same, u-strings do not exist in python 3 anymore, but literal (r-strings) need to be indicated

File: mriqc/workflows/functional.py [hunk 3]
```diff
@@ -142,6 +142,15 @@ def fmri_qc_workflow(name='fMRIQC', settings=None):
         workflow.connect(bmw, 'outputnode.out_file', plot_mean, 'in_mask')
         workflow.connect(bmw, 'outputnode.out_file', plot_tsnr, 'in_mask')
 
+    # Save the TSNR results (tsnr, mean, stdev)
+    tsnr_results = pe.Node(nio.DataSink(parameterization=False), name='tsnr_results')
```
Comment (oesteban) [inline mriqc/workflows/functional.py]: LGTM

File: mriqc/reports/generators.py (hunk 1 above)
Comment (jvarada) [inline mriqc/reports/generators.py]: Is there a reason you want them to stay as a raw string though? In python 3 (and in py2 with the unicode_literals import), it should handle the unicode characters with the backslash correctly without having to make it a raw string

Comment (oesteban) [inline mriqc/reports/generators.py]: You are right. That r needs also to go away. Sorry

**Comment by oesteban** [comment]
@jvarada thanks for this PR, it was very useful today in one little project we have here!

**Pull request merged**